inc/sailag.h

```diff
@@ -141,6 +141,15 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_DROP_TAGGED,
 
+    /**
+     * @brief LAG Name
+     *
+     * @type char
+     * @flags CREATE_ONLY
+     * @default \"\"
```

Comment:
you don't need to add '\'

Reply:
It would build fail if I don't add \"\".
build fail message:
ERROR: invalid default tag value '' on sai_lag_attr_t SAI_LAG_ATTR_NAME

Reply:
ok, leave it as is, after you merge that i will work on fixing parser to allow ""